apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTiming.cs

```diff
@@ -0,0 +1,109 @@
+    /// <returns>A new instance based on the existing values, but with the specified initial backoff.</returns>
+    public RetryTiming WithInitialBackoff(TimeSpan initialBackoff) =>
+        new(initialBackoff: initialBackoff, maxBackoff: MaxBackoff, backoffMultiplier: BackoffMultiplier);
+
+    /// <summary>
+    /// Returns the existing retry timings configurations using the specified maximum backoff.
+    /// </summary>
+    /// <param name="maxBackoff">The new value for <see cref="MaxBackoff"/>.</param>
+    /// <returns>A new instance based on the existing values, but with the specified max backoff.</returns>
+    public RetryTiming WithMaxBackoff(TimeSpan maxBackoff) =>
+         new(initialBackoff: InitialBackoff, maxBackoff: maxBackoff, backoffMultiplier: BackoffMultiplier);
+
+    /// <summary>
+    /// Returns the existing retry timings configurations using the specified backoff multiplier.
+    /// </summary>
+    /// <param name="backoffMultiplier">The new value for <see cref="BackoffMultiplier"/>.</param>
+    /// <returns>A new instance based on the existing values, but with the specified backoff multiplier.</returns>
+    public RetryTiming WithBackoffMultiplier(double backoffMultiplier) =>
+        new(initialBackoff: InitialBackoff, maxBackoff: MaxBackoff, backoffMultiplier: backoffMultiplier);
+
+    private RetryTiming()
+    {
+        InitialBackoff = RetryTiming.DefaultInitialBackoff;
+        MaxBackoff = RetryTiming.DefaultMaxBackoff;
+        BackoffMultiplier = RetryTiming.DefaultBackoffMultiplier;
+    }
+
+    /// <summary>
+    /// Creates an instance with the specified values.
+    /// </summary>
+    /// <param name="initialBackoff">Initial backoff for retry. It must be non-negative.</param>
+    /// <param name="maxBackoff">Maximum backoff for retry. This is never smaller than <paramref name="initialBackoff" />.</param>
+    /// <param name="backoffMultiplier">Backoff multiplier for retry. It is always greater than or equal to 1.0.</param>
+    public RetryTiming(TimeSpan initialBackoff, TimeSpan maxBackoff, double backoffMultiplier)
+    {
+        InitialBackoff = initialBackoff >= TimeSpan.Zero ? initialBackoff : throw new ArgumentOutOfRangeException(nameof(initialBackoff), $"'{InitialBackoff}' parameter value must be non-negative.");
+        MaxBackoff = maxBackoff >= initialBackoff ? maxBackoff : throw new ArgumentOutOfRangeException(nameof(maxBackoff), $"{MaxBackoff} parameter value must be at least as long as initialBackoff.");
+        BackoffMultiplier = backoffMultiplier >= 1 ? backoffMultiplier : throw new ArgumentOutOfRangeException(nameof(backoffMultiplier), $"{BackoffMultiplier} parameter value must at least be 1.");
+    }
+
+    internal TimeSpan GetDelay(int failureCount)
+    {
+        TimeSpan delay = failureCount > 1 ? InitialBackoff + TimeSpan.FromSeconds(Math.Pow(2.0, failureCount - 2) * BackoffMultiplier) : InitialBackoff;
```

Comment:
As noted earlier, this isn't correct. Please make sure you understand the expected meaning of BackoffMultiplier, write some tests, and then fix them appropriately.